libass/ass_blur.c

```diff
@@ -75,10 +72,10 @@ inline static void copy_line(int16_t *buf, const int16_t *ptr, uintptr_t offs, u
 void ass_stripe_unpack_c(int16_t *dst, const uint8_t *src, ptrdiff_t src_stride,
                          uintptr_t width, uintptr_t height)
 {
-    for (uintptr_t y = 0; y < height; ++y) {
+    for (uintptr_t y = 0; y < height; y++) {
```

Comment:
Would be nice to split these little stylistic changes into a separate commit.

Reply:
Done